src/api/json/catalog.json

```diff
@@ -222,6 +222,11 @@
         "draft-01": "https://carafe.fm/schema/draft-01/bundle.schema.json"
       }
     },
+    {
+      "name": "CityJSON",
+      "description": "Schema for the representation of 3D city models",
+      "url": "https://github.com/cityjson/specs/blob/master/schemas/cityjson.min.schema.json"
```

Comment:
It should be the raw url (https://raw.githubusercontent.com/cityjson/specs/master/schemas/cityjson.min.schema.json)